Correct workflow badge 'event' example (#12331)
github/docs: content/actions/monitoring-and-troubleshooting-workflows/adding-a-workflow-status-badge.md

![example branch parameter](https://github.com/github/docs/actions/workflows/main.yml/badge.svg?branch=feature-1)
```

> ## Using the `on` parameter

Reviewer: ```suggestion
## Using the `event` parameter
```

> This Markdown example adds a badge that displays the status of workflow runs triggered by the `pull_request` event.

Reviewer: ```suggestion
This Markdown example adds a badge that displays the status of continuous integration builds in your default branch; those are builds that are triggered by the `push` event (when a commit is pushed or when a pull request is merged) into the repository's default branch.
```

```markdown

> ![example on parameter](https://github.com/github/docs/actions/workflows/main.yml/badge.svg?on=pull_request)

Reviewer: ```suggestion
![example on parameter](https://github.com/github/docs/actions/workflows/main.yml/badge.svg?event=push)
```
